x-pack/plugins/ml/public/application/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -93,48 +89,82 @@ export const AnalyticsPanel: FC<Props> = ({
 
   const panelClass = isInitialized === false ? 'mlOverviewPanel__isLoading' : 'mlOverviewPanel';
 
+  const transformsLink = `${basePath.get()}/app/management/data/transform`;
+
+  const noDFAJobs = errorMessage === undefined && isInitialized === true && analytics.length === 0;
+
+  if (noDFAJobs) {
+    return (
+      <EuiEmptyPrompt
+        layout="horizontal"
+        hasBorder={true}
+        hasShadow={false}
+        icon={<EuiImage size="fullWidth" src={adImage} alt="anomaly_detection" />}
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.ml.overview.analyticsList.createFirstJobMessage"
+              defaultMessage="Create your first data frame analytics job"
+            />
+          </h2>
+        }
+        body={
+          <>
+            <p>
+              <FormattedMessage
+                id="xpack.ml.overview.analyticsList.emptyPromptText"
+                defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis on your data and annotates it with the results. The job puts the annotated data and a copy of the source data in a new index."
```

Comment:
I think it would be nice to allude to the inference possibilities in this description. For example:
```suggestion
                defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis and put the annotated data in a new index. You can also use the classification or regression trained models against new data in pipelines or aggregations."
```

Reply:
Changed in 3f320b162d8391d05afce5907875b33e9fef7fe3